### DISCUSSION TITLE
searchable() - dropdown lagging, refeshing the page there is lag and within sec it correct itself

### DISCUSSION THREAD

**Discussion by Suraj127-git** [filamentphp/filament#11085] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

I have images that show it takes 5 sec to get it correct dropdown and I tried many cache command

php artisan view: cache php artisan route: cache php artisan config: cache php artisan icons: cache

This is the code

Forms\Components\Section::make('Address Details') ->schema([ Forms\Components\Select::make('country_id') ->relationship(name: 'country', titleAttribute: 'name') ->searchable() ->preload() ->multiple() ->required(), Forms\Components\Select::make('state_id') ->relationship(name: 'state', titleAttribute: 'name') ->searchable() ->preload() ->multiple() ->required(), Forms\Components\Select::make('city_id') ->relationship(name: 'city', titleAttribute: 'name') ->searchable() ->preload() ->multiple() ->required(), Forms\Components\Select::make('department_id') ->relationship(name: 'department', titleAttribute: 'name') ->searchable() ->preload() ->multiple() ->required(), ])->columns(3),

There are 2 images attached their image where it take time to load or lag and in 5 seconds then it corrected itself

![filament-error](https://github.com/filamentphp/filament/assets/52707003/3f35c906-f1fd-4b78-ac11-29c0bce177f5)
![filament-succes](https://github.com/filamentphp/filament/assets/52707003/91047688-fc21-48d3-80fa-b0d481f701d5)

Reproduction repository
https://github.com/Suraj127-git/laravel-filament-panel

**Comment by leandrocfe** [comment]
Did you try it without the debugbar?

**Reply by Suraj127-git** [reply]
Yes, I did then too it showing like that

**Comment by Suraj127-git** [comment]
I installed and ran OCTANE_SERVER=frankenphp, and it greatly improved the speed of my application. However, it still has an issue where it takes around 1 second to load, which is noticeable to the human eye. I am therefore looking for an expert in Filament to investigate this matter.

**Reply by leandrocfe** [reply]
Share the project on the github. I can take a look
